### DISCUSSION TITLE
error getting bundle info for crc_libvirt_4.11.7_amd64.crcbundle

### DISCUSSION THREAD

**Discussion by cgleiyucun** [okd-project/okd#1383] [General]
DEBU error getting bundle info for crc_libvirt_4.11.7_amd64.crcbundle: error reading /home/lyc/.crc/cache/crc_libvirt_4.11.7_amd64/crc-bundle-info.json file: open /home/lyc/.crc/cache/crc_libvirt_4.11.7_amd64/crc-bundle-info.json: no such file or directory 
DEBU error reading /home/lyc/.crc/cache/crc_libvirt_4.11.7_amd64/crc-bundle-info.json file: open /home/lyc/.crc/cache/crc_libvirt_4.11.7_amd64/crc-bundle-info.json: no such file or directory 
INFO Getting bundle for the CRC executable        
DEBU Ensuring directory /home/lyc/.crc/cache exists 
DEBU Got bundle path: /home/lyc/.crc/cache/crc_libvirt_4.11.7_amd64.crcbundle 
DEBU Downloading https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/openshift/4.11.7/crc_libvirt_4.11.7_amd64.crcbundle to /home/lyc/.crc/cache/crc_libvirt_4.11.7_amd64.crcbundle 
server returned 307 Temporary Redirect

**Comment by cgleiyucun** [comment]
![1667275234976](https://user-images.githubusercontent.com/42564884/199155907-2f1bba51-9b20-40a3-a18f-49dad50ab822.png)

**Comment by cgleiyucun** [comment]
no crc-bundle-info.json

**Comment by vrutkovs** [comment]
See https://github.com/okd-project/okd.io/pull/7/files
